### PR TITLE
Pager - Add a form to go directly to a specific page

### PR DESCRIPTION
**Pull request by SimonMellerin**

### CODE REVIEW

**Comment by pounard** [comment]
Mostly all good, except that I don't see any usage of this new function ? Is it for some project specific use case ? Would it worth appending an option somewhere to automatically embed it in default tables/pages ?

**Comment by SimonMellerin** [comment]
It will automatically appears add the end of the pager : line 247 of calista-twig/templates/page/page.html.twig

doing something like : 
image

**Comment by pounard** [comment]
Ok I'd prefer not to alter the default rendering, but having an option to make it appear would be nice.

File: calista-twig/templates/page/page.html.twig [hunk 1]
```diff
@@ -202,12 +217,12 @@
         {% if 1 < page %}
           <li class="page-item">
             <a class="page-link" href="{% with {index: 1} %}{{ block('pager_href') }}{% endwith %}" aria-label="{{ "First"|trans }}" data-page-link>
-              <span aria-hidden="true">&larr;</span>
+              <span aria-hidden="true">&#x21E4;</span>
```
[the remark this answers is not in the document]
Comment (SimonMellerin) [reply calista-twig/templates/page/page.html.twig:220]: No, like I was updating the pager, I took the liberty to update those arrows. 

From :
image

To :
image

I found the first double arrow from the original pager ambiguous, it looks like a fast forward while it goes only to next page

[the remark this answers is not in the document]
Comment (pounard) [reply calista-twig/templates/page/page.html.twig:219]: OK, fine.

**Pull request merged**